conda/recipes/libcudf/conda_build_config.yaml

```diff
@@ -0,0 +1,14 @@
+cmake_version:
+  - ">=3.20.1"
```

Comment:
Should these versions go into a versions.yaml file (as in the integration repo) since you'll want these to sync up with what's in the meta.yaml?

Reply:
It's an interesting question. Have wondered if we need a `rapids-pinning` recipe/package, which includes a bunch of common pinnings that are used throughout to build recipes. Maybe that is a separate thing to tackle though

Reply:
By default, `conda` will recognize a `conda_build_config.yaml` file in the recipe directory ([src](https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#creating-conda-build-variant-config-files)). In the case of the integration repo, the `conda build` command uses the `--variant-config-files`/`-m` flag to indicate that the `versions.yaml` file should be used as a `conda_build_config.yaml` file ([src](https://github.com/rapidsai/integration/blob/b1deb459fb71d37aaaab50a8f0deaa10b0197383/ci/cpu/build.sh#L56)).

Reply:
@jakirkham's point about sharing the pinnings across RAPIDS libraries is potentially worth thinking about (I think a l lot of our `ci` and `conda` directories can be shared across RAPIDS packages), but that's probably out of scope for this PR. Thanks for clarifying that this file already serves that purpose.